docs/sources/k6/next/javascript-api/k6-browser/metricmessage.md

```diff
@@ -0,0 +1,60 @@
+---
+title: 'MetricMessage'
+slug: 'metricmessage'
+description: 'Browser module: MetricMessage Object'
+weight: 03
+---
+
+# MetricMessage
+
+A `MetricMessage` object allows tagging of metrics that are measured and emitted for the page.
+
+`MetricMessage` objects are dispatched by the page when a handler is registered with [page.on('metric')](https://grafana.com/docs/k6/<K6_VERSION>/javascript-api/k6-browser/page/on). For each metric that is measured and emitted for the page, the k6 browser delivers a `MetricMessage` object to the registered handlers, allowing them to pattern match and tag metrics.
+
+## tag
+
+The `tag` method matches the given `matches` with the current metric's url and name tags. When a match is found, it will use `name` to replace the existing url and name tag values.
+
+Doing this helps to group metrics with disparate url and name tags, which are, in fact, referencing the same resource so that a correlation can be found over time and to reduce the cardinality of the metrics.
+
+<TableWithNestedRows>
+
+| Parameter               | Type     | Description                                                                                                                                                                                                                                                                                 |
+| ----------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| tagMatch                | object   | It is required.                                                                                                                                                                                                                                                                             |
+| tagMatch.name           | string   | It is a required field and it must not be an empty string. It will replace the current metric's url and name tag values when a match is found.                                                                                                                                              |
```

Comment:
I'm a little bit confused by this description and the following ones as well. 🤔 

What happens with the value of this property? I'm wondering if we could tweak the description for this and the following values to be something along the lines of: "The name value that replaces the current metric's URL and name tag values, if a match is found. Required, and must not be an empty string."

Reply:
I've made the change, WDYT? 152d0b3df5d63a1969cd863a54886a16170f9801